### ISSUE TITLE
`missing parent call` for static parent getters

### ISSUE THREAD

**Issue by vladyslavstartsev** [kalessil/phpinspectionsea#521]
Sometimes it's irrelevant because plugin checks, for example, for `getOptions` who's parent class has an empty array.
The example.
Parent class.
![image](https://user-images.githubusercontent.com/17382248/30502901-d341b564-9a70-11e7-9f3e-9d6aa8e05091.png)
Child class.
![image](https://user-images.githubusercontent.com/17382248/30502911-d8a06442-9a70-11e7-8a52-80b41cff1f7e.png)

I mean if I'll call `parent::getOptions()`, nothing will change

P.S. `static parent getters` is that the right name for it?

**Comment by kalessil** [comment]
Not really getting the point: do we produce false positives, do we need to report something or it's something else?

**Comment by vladyslavstartsev** [comment]
@kalessil, yes, it's kinda false positives,nothing needed o be reported. 
I mean I get `Missing parent call`  when the mouse pointer is moved on method name. But I will call it it will not change anything.

Example.
Parent class
```php
    protected function getOptions()
    {
        return [
          ['filename', 'F', InputOption::VALUE_OPTIONAL, 'The path to the helper file', $this->filename],
          ['dir', 'D', InputOption::VALUE_OPTIONAL | InputOption::VALUE_IS_ARRAY, 'The model dir', []],
          ['write', 'W', InputOption::VALUE_NONE, 'Write to Model file'],
          ['nowrite', 'N', InputOption::VALUE_NONE, 'Don\'t write to Model file'],
          ['reset', 'R', InputOption::VALUE_NONE, 'Remove the original phpdocs instead of appending'],
          ['ignore', 'I', InputOption::VALUE_OPTIONAL, 'Which models to ignore', ''],
        ];
    }
```
Child class
```php
    protected function getOptions()
    {
        return [];
    }
```

I'll do something like [here](https://3v4l.org/h0h03), it will not make sense because parent class method is an empty array.
What I want (or think it will be good one) is nothing should be suggested in this situation

**Comment by kalessil** [comment]
@vladyslavstartsev : this reported by `Missing parent call for method` inspection by PhpStorm itself, not by Php Inspections (EA Extended).

**Comment by vladyslavstartsev** [comment]
@kalessil, oh, got it
